explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -40,6 +45,12 @@ export function TransactionDetailsPage({ signature: raw }: Props) {
     }
   } catch (err) {}
 
+  const status = useTransactionStatus(signature);
+
+  const autoRefreshInProcess = !(
+    status?.data?.info && status.data.info.confirmations === "max"
```

Comment:
What happens if a transaction is never confirmed?

Reply:
I accidentally factored out the case where transaction isn't found in this (this has been updated). If a transaction is never confirmed, I suppose it would keep refreshing forever.

Reply:
Cool, thanks for fixing that! So if a transaction lands in a fork, it appears that it will always stay stuck at 0 confirmations until it's purged from the node's status cache: https://github.com/solana-labs/solana/blob/dbd079f54ce5fcb548358ffa669963952312d539/core/src/rpc.rs#L786

Rather than refreshing forever how about bailing after a few retries if confirmations is stuck at 0?